### ISSUE TITLE
More recipes for blast furnace

### ISSUE THREAD

**Issue by dnarik** [GregTechCE/GregTech#1041]
tl;dr at the bottom

I would love to see some historical recipes for blast furnace: decomposing stuff like tetrahedrite and bauxite for extra metals yields, but no oxygen/hydrogen/sulfur, or maybe in a different form, requiring further purification (btw all the dusts and liquids are already in game), using cryolite to get aluminium, maybe even add complexity (or just options to get more ingots) to creating steel. Also, some types of ores, mainly gravel ones, don't have recipes in the blast furnace, even if they should (i.e. limonites)

consider adding more recipes for blast furnaces, I'd love to give some suggestions if that's ok (and sources mentioning them, if needed)
